docker/Dockerfile.aarch64-linux-android

```diff
@@ -20,7 +24,7 @@ RUN /android-system.sh arm64
 COPY qemu.sh /
 RUN /qemu.sh aarch64
```

Comment:
is it possible to move this back? That way when changing the args you don't have to rebuild qemu

Reply:
Sure, it is.